### PR TITLE
proposing some changes to lfdnoise3 documentation

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**
proposing some changes to lfdnoise3 documentation
adding examples from @ woutersnoei canto ostinato classes
hope wouter doesn't mind at all
looking forward for your feedback of the code i've written (or copied)
hope this is helpful for the community.
kind regards

### CODE REVIEW

**Comment by snappizz** [comment]
The new examples use the class RoundPan, which is from the wslib quark.

**Comment by tiagmoraismorgado** [comment]
tomorrow i will replace it by pan2.ar, pan4.ar and pan8.ar to use it as examples. today i am already pretty tired and i might be about to go to bed

**Comment by tiagmoraismorgado** [comment]
i made a slickly review to this, but it seems to not be running properly, the examples, and needing some tweaking mostly about deprecated methods, and stuff from external libraries. if someone can help me with this task i would be thankful

File: HelpSource/Classes/LFDNoise3.schelp [hunk 1]
```diff
@@ -42,40 +42,17 @@ Examples::
...
-		SinOsc.ar(
-			LFDNoise3.ar(4, 400, 450),
-			0, 0.2
-		)
-}.play;
-)
-
-////////////////////////////////
-////////////////////////////////
+({SinOsc.ar(LFDNoise3.ar(4, 400, 450), 0, 0.2)}.play;)
```
Comment (telephon) [inline HelpSource/Classes/LFDNoise3.schelp]: no need for the round parentheses

File: HelpSource/Classes/LFDNoise3.schelp [hunk 2]
```diff
@@ -109,17 +86,13 @@ SynthDef( "canto_revDelay", { |in = 0, private = 0, amp = 0.1,
 	});
 
 	out = out.collect({ |sig, i|
-		RoundPan.ar( ~canto.numChan ? 4, sig, LFDNoise3.kr(0.05, 2) );
-	}).sum;
-
+		Pan4.ar(sig, trig * 0.25, LFDNoise3.kr(modSpeed) * pan, trig * 0.25, LFDNoise1.kr(modSpeed) * pan)});
```
Comment (telephon) [inline HelpSource/Classes/LFDNoise3.schelp]: `Pan4.ar` has the follwowing arguments: `in, xpos = 0.0, ypos = 0.0, level = 1.0;` I'm not sure what you are trying to do here.

Why not `Pan2.ar( sig, LFDNoise3.kr(0.05) );` ?

File: HelpSource/Classes/LFDNoise3.schelp [hunk 3]
```diff
@@ -151,24 +124,18 @@ SynthDef( "canto_revDelay_quadIn", { |in = 0, private = 0, amp = 0.1,
 	});
 
 	out = out.collect({ |sig, i|
-		RoundPan.ar( ~canto.numChan ? 4,
-			LFDNoise3.kr(0.2).range(-0.5,1).max(0) * sig,
-			LFDNoise3.kr(0.05, 2) );
+	Pan4.ar(sig, trig * 0.25, LFDNoise3.kr(modSpeed) * pan, trig * 0.25, LFDNoise1.kr(modSpeed) * pan);
```
Comment (telephon) [inline HelpSource/Classes/LFDNoise3.schelp]: same here.

File: HelpSource/Classes/LFDNoise3.schelp [hunk 4]
```diff
@@ -179,17 +146,13 @@ SynthDef( "bassdrum", { |amp = 0.25, freq = 42, pulse = 0.1, attackFreq = 100, d
 			LFDNoise3.kr(0.1).exprange(-0.1.midiratio, 0.1.midiratio),
 		-1);
 	sig = sig * Env([1,0], [sustain], -4).kr(2);
-	sig = RoundPan.ar( 4, sig, pan + Rand(-0.1,0.1)) * amp;
+	sig = Pan4.ar(sig, trig * 0.25, LFDNoise3.kr(modSpeed) * pan, trig * 0.25, LFDNoise1.kr(modSpeed) * pan);
```
Comment (telephon) [inline HelpSource/Classes/LFDNoise3.schelp]: … and here.

File: HelpSource/Classes/LFDNoise3.schelp [hunk 5]
```diff
@@ -151,24 +124,18 @@ SynthDef( "canto_revDelay_quadIn", { |in = 0, private = 0, amp = 0.1,
...
-		RoundPan.ar( ~canto.numChan ? 4,
-			LFDNoise3.kr(0.2).range(-0.5,1).max(0) * sig,
-			LFDNoise3.kr(0.05, 2) );
+	Pan4.ar(sig, trig * 0.25, LFDNoise3.kr(modSpeed) * pan, trig * 0.25, LFDNoise1.kr(modSpeed) * pan);
 	}).sum;
 
 	out = RemoveBadValues.ar( out );
 
 	Out.ar(0, Env.cutoff(0.1).kr(2,gate) * out * amp * 2);
 }).load(s);
```
Comment (telephon) [inline HelpSource/Classes/LFDNoise3.schelp]: use `add` instead of `load(s)` if you can.

File: HelpSource/Classes/LFDNoise3.schelp [hunk 6]
```diff
@@ -345,7 +300,7 @@ SynthDef( "canto_noteNoise", { |amp = 0.1, sustain = 1, damp = 16, freq = 440, s
 
 	sig = BPeakEQ.ar( sig, 1000, 1, mod * -3 );
 	sig = BLowCut.ar( sig, hpf, 2 );
-	sig = RoundPan.ar(~canto.numChan ? 4, sig, Rand(-0.05,0.05) + pan);
+	sig = Pan4.ar(sig, trig * 0.25, LFDNoise3.kr(modSpeed) * pan, trig * 0.25, LFDNoise1.kr(modSpeed) * pan);
 	env = Env([1,1,0], [sustain * sharpness.linlin(16,32,1,0.01).clip(0.01,1), 0.1]).ar(2);
 	sig = RemoveBadValues.ar(sig);
 	OffsetOut.ar( 0, sig * env * amp );
```
Comment (telephon) [inline HelpSource/Classes/LFDNoise3.schelp]: it is better style for SynthDefs to use an `out` argument instead of 0.

**Comment by vivid-synth** [comment]
I like these examples but I think they're too complex for a helpfile: it's really not clear what part LFDNoise3 is playing in creating the sound.

If we were to use them too we should definitely ask Wouter's permission.

**Pull request closed without merging**